### ISSUE TITLE
Authentication Possible without Submitting or Reviewing of any Documents by the new users

### ISSUE THREAD

**Issue by KarthikVV24-11-1999** [gcivil-nyu-org/fall24-monday-team4#114]
Accounts Used: Karthik, test2
Bug: Authentication possible without submitting documents
Steps Followed:
1. Sign up for a new account and login into it
2. Do not upload any documents as prompted, and logout immediately
3. Login into an admin account
4. Ensure that no new documents are uploaded for this new User in the User Documents Tab 
5. Search for the New User in the Manage Users tab and Authenticate Him/Her 
6. Check the email for the User to receive an email that says that "Congratulations! Our team has reviewed your submitted documents, and we are delighted to inform you that your account has been successfully verified. You can now log in and enjoy the full range of services that RoutePals provides"

Expected Results: Admin should not be allowed to authenticate the user without reviewing any submitted documents

[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd3pjQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a28b5c7ccc5d8d36bb8bf80e712fb1f58d3463a0/Screen%20Recording%202024-11-22%20at%2012.07.01%E2%80%AFAM.mov)

**Comment by shashankdatta** [comment]
Not a bug, thats the functionality, admin has all the rights to approve a user without a doc too. 

But thank you for bringing this to our attention

**Comment by KarthikVV24-11-1999** [comment]
Again, it's an enhancement that You might want to consider here, because an admin, even though is of greater privilege in the application, should not have full rights to accept anyone as a user, without a recorded proof, especially because there maybe tens, if not hundreds, of admins to go through the application documents by an enormously large inflow of users, when the app really grows in scale, and not all of them may be actually trustworthy
So, hence, to maintain the legitimacy of the application, I believe, letting users in without verification is not optimal

**Comment by shashankdatta** [comment]
True, but we are keeping the features simple and testable without the need
for students to upload their personal files given the S3 free usage
constraints. We just gave an insight of what could be a great feature but
not necessarily a strict guardrail.

Many of our inspirations were from successful educational platforms like
Coursera and edX started with basic verification before evolving to more
stringent systems as they scaled.

Nevertheless, our django admin portal system already includes logs of the
documents that are uploaded and when so we can def track the users pretty
quickly, which helps maintain accountability even without document
verification.

But I get your point, we will definitely consider it if it deems necessary
as we reach the final stages of the project. Thank you for the peer review
though, really appreciate it

**Comment by KarthikVV24-11-1999** [comment]
Thanks for your clarifications and insights
Appreciate the effort in due diligence
